### PR TITLE
[SMALLFIX] Update outdated version in README

### PR DESCRIPTION
**Pull request by ifcharming**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ The master branch is in version 1.7.0-SNAPSHOT:
...
 
-As of 1.5.0, Alluxio provides several different client artifacts. The Alluxio file system interface
+As of 1.6.0, Alluxio provides several different client artifacts. The Alluxio file system interface
 provided by the `alluxio-core-client-fs` artifact is recommended for the best performance and access
 to Alluxio-specific functionality. If you want to use other interfaces, include the appropriate
 client artifact. For example, `alluxio-core-client-hdfs` provides a client implementing HDFS's file
 system API.
 
-For Alluxio versions below 1.5.0, use the `alluxio-core-client` artifact.
+For Alluxio versions below 1.6.0, use the `alluxio-core-client` artifact.
```
Comment (apc999) [inline README.md]: I don't think we should change here.  In Alluxio 1.5, we should refer to `alluxio-core-client-fs` or `alluxio-core-client-hdfs`

File: README.md [hunk 2]
```diff
@@ -20,13 +20,13 @@ The master branch is in version 1.7.0-SNAPSHOT:
 
 
 
-As of 1.5.0, Alluxio provides several different client artifacts. The Alluxio file system interface
+As of 1.6.0, Alluxio provides several different client artifacts. The Alluxio file system interface
```
Comment (apc999) [inline README.md]: same, I don't think we should change the version here. Same reason below

File: README.md (hunk 1 above)
Comment (apc999) [reply README.md]: maybe rephrase to "For Alluxio of version 1.4 or earlier"

Comment (ifcharming) [reply README.md]: then shall we include both 1.5 and 1.6 since the artifact path is different?

Comment (ifcharming) [reply README.md]: done, thanks!

**Comment by ifcharming** [comment]
@apc999 can you PTAL? thanks
